crates/bevy_render/src/renderer/mod.rs

```diff
@@ -209,6 +209,15 @@ pub async fn initialize_renderer(
             // integrated GPUs.
             features -= wgpu::Features::MAPPABLE_PRIMARY_BUFFERS;
         }
+
+        // RAY_QUERY and RAY_TRACING_ACCELERATION STRUCTURE will sometimes cause DeviceLost failures on platforms
+        // that report them as supported:
+        // <https://github.com/gfx-rs/wgpu/issues/5488>
+        // WGPU also currently doesn't actually support these features yet, so we should disable
+        // them until they are safe to enable.
+        features -= wgpu::Features::RAY_QUERY;
+        features -= wgpu::Features::RAY_TRACING_ACCELERATION_STRUCTURE;
+
```

Comment:
This is ok, but maybe the approach for overrides like this should be to add them to the WgpuSettings disabled_features default so that people can still override it if they know better?

Reply:
This would mean that setting `disabled_features` could inadvertently overwrite the default overrides. I think its best to default that to none.

Users can still set these if they set `WgpuSettings::features`, which feels like the better pattern.